### PR TITLE
Update datasets_icons.md

### PR DESCRIPTION
**Pull request by jennaj**
Adding in more icons present in the updated history panel view.

Some may need to be added to GTN.

In progress .. would like to capture multi-select icons too. This is just a start.

### CODE REVIEW

**Comment by jennaj** [comment]
Let's push this next week if possible and worry about the other icons later. All of those included here should be supported (passed tests)

File: faqs/galaxy/datasets_icons.md [hunk 1]
```diff
@@ -9,9 +9,14 @@ contributors: [jennaj, garimavs]
...
-- {% icon galaxy-pencil %} **"Pencil icon"**: Edit attributes of the job.
-- {% icon galaxy-delete %} **"Trash icon"**: Delete the job.
-- {% icon galaxy-info %} **"Info icon"**: Job details and run information.
-- {% icon galaxy-refresh %} **"Refresh/Rerun icon"**: Run this (selected) job again or examine original submitted form (filled in).
-- {% icon galaxy-bug %} **"Bug icon"**: Review and optionally submit a bug report.
+- {% icon galaxy-eye %} **"Eye icon"**: Display dataset contents.
+- {% icon galaxy-pencil %} **"Pencil icon"**: Edit attributes of dataset metadata: labels, datatype, database.
+- {% icon galaxy-delete %} **"Trash icon"**: Delete the dataset.
+- {% icon fa-save %} **"Disc icon"**: Download the dataset.
+- {% icon link %} **"Copy link"**: Copy link URL to the dataset.
```
Comment (hexylena) [inline faqs/galaxy/datasets_icons.md:16]:
```suggestion
- {% icon galaxy-save %} **"Disc icon"**: Download the dataset.
- {% icon galaxy-link %} **"Copy link"**: Copy link URL to the dataset.
```

File: faqs/galaxy/datasets_icons.md [hunk 2]
```diff
@@ -9,9 +9,14 @@ contributors: [jennaj, garimavs]
...
+- {% icon galaxy-info %} **"Info icon"**: Dataset details and job runtime information: inputs, parameters, logs.
+- {% icon galaxy-refresh %} **"Refresh/Rerun icon"**: Run this (selected) job again or examine original submitted form.
+- {% icon fa-bar-chart-o %} **"Visualize icon"**: External display links (UCSC, IGV, NPL, PV); Charts and graphing; Editor (manually edit text).
```
Comment (hexylena) [inline faqs/galaxy/datasets_icons.md:19]:
```suggestion
- {% icon galaxy-barchart %} **"Visualize icon"**: External display links (UCSC, IGV, NPL, PV); Charts and graphing; Editor (manually edit text).
```

File: faqs/galaxy/datasets_icons.md [hunk 3]
```diff
@@ -9,9 +9,14 @@ contributors: [jennaj, garimavs]
...
+- {% icon fa-sitemap %} **"Dataset Map icon"**: Filter the history for related Input/Output Datasets.
+- {% icon fa-sitemap %} **"Selected Dataset Map icon"**: The "root dataset" for Input/Output Map. Click to clear filter.
```
Comment (hexylena) [inline faqs/galaxy/datasets_icons.md:21]: We should add these two to `_config.yml` with some nicer alias, `galaxy-map-datasets` maybe?

**Comment by shiltemann** [comment]
thanks @jennaj ! :)

**Pull request merged**